airflow-core/src/airflow/api_fastapi/execution_api/routes/task_instances.py

```diff
@@ -70,7 +70,7 @@
 ti_id_router = VersionedAPIRouter(
     dependencies=[
         # This checks that the UUID in the url matches the one in the token for us.
```

Comment:
Lets remove this

Reply:
that is still valid though, I know it is duplicated but easy to understand from that comment than looking at defintion

Reply:
Cool